theme-default/gulp/config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "project":{
+        "name" : "dna-recipe",
```

Comment:
This config is normally kept outside the project root (because it can be different for every dev). Its currently named `_npm_environment.json`